app/models/certificate.server.ts

```diff
@@ -44,3 +44,7 @@ export function updateCertificateById(
 export function deleteCertificateById(id: Certificate['id']) {
   return prisma.certificate.delete({ where: { id } });
 }
+
+export function getCertificateCount() {
+  return prisma.certificate.count();
```

Comment:
this is the value that I show to the admin as the total number of certificates in the system. I think it can be confusing for the admin to see the number that wouldn't reflect the real number of certificates we have in DB. I am curious what you think. @humphd

Reply:
What if you do an icon here.  If the user has one or more [issued](https://github.com/DevelopingSpace/starchart/blob/main/prisma/schema.prisma#L89), non-expired certificates, show one icon.  If they have none, show another.  If they have one, but it's expired, show a third?

Could do this in a follow-up so we don't block this.

Reply:
I am sorry, but I feel like we have a little misunderstanding over here. I am using the `getCertificateCount` only to show the total number of certificates in the system on the metric card. I don't use it for checking the certificates of a specific users, for this purpose I use `getIssuedCertificateByUsername` for each user individually 

![image](https://user-images.githubusercontent.com/32075241/228581763-438919d3-5185-42d6-ba90-936ab6b1aa53.png)

Reply:
Got it!